hbase-compression/hbase-compression-snappy/src/test/java/org/apache/hadoop/hbase/io/compress/xerial/TestHFileCompressionSnappy.java

```diff
@@ -37,10 +39,15 @@ public class TestHFileCompressionSnappy extends HFileTestBase {
   public static final HBaseClassTestRule CLASS_RULE =
     HBaseClassTestRule.forClass(TestHFileCompressionSnappy.class);
 
+  private static final Logger LOG = LoggerFactory.getLogger(TestHFileCompressionSnappy.class);
   private static Configuration conf;
 
   @BeforeClass
   public static void setUpBeforeClass() throws Exception {
+    if (!SnappyCodec.isLoaded()) {
```

Comment:
We could use Assume.assumeTrue here to skip the test.

Reply:
Thanks. I did not know about that. Will use it.